docs/reference-guide/modules/events/pages/event-processors/streaming.adoc

```diff
@@ -1795,8 +1531,8 @@ The type of the `resetContext` has to match the parameter's, or it will be null.
 == Multiple event sources
```

Comment:
Support for the `MultiStreamableEventSource` is to be implemented in 5.2.0. So, I think we should omit this entire section for now.